package.json

```diff
@@ -65,7 +65,8 @@
     "test": "npx mocha -r ts-node/register --extensions ts 'test/**/*.{test,spec}.ts'"
   },
   "lint-staged": {
-    "*.ts": "eslint --cache --fix",
+    "Readme.md": "npx embedme Readme.md",
```

Comment:
Should we include `embedme` as a `devDependency`, shouldn't we?

Reply:
I think we should run the command even if the config files change, and not only if readme changes.

Reply:
oh yea I'll actually change it to run it every time (ie match any change) just to make sure that whatever files we might want to embed in the future will get embedded without changing this config.

Reply:
as to adding that to dependency, I don't really see any benefit in that, but will do if you think it would add value.

Reply:
added to dev deps